### PR TITLE
Add zipl and grub2 CPEs

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Add zipl and grub2 CPEs
- Update relevant CPE-Dictionaries

#### Rationale:
- This will make it possible to distinguish between GRUB2 and zIPL rule.

### CODE REVIEW

**Comment by mildas** [comment]
**Changes identified:**
Rule checks:
 OVAL check is newly added.
 The rule doesn't occur in any profile nor product.
Others:
 Python abstract syntax tree change found in ssg/constants.py.

**Recommended tests to execute:**
 (cd build && cmake ../ && ctest -j4)

File: linux_os/guide/system/bootloader-zipl/group.yml [hunk 1]
```diff
@@ -8,4 +8,4 @@ description: |-
     options to it.
     The default {{{ full_name }}} boot loader for s390x systems is called zIPL.
 
-platform: machine
+platform: zipl
```
Comment (yuumasato) [inline linux_os/guide/system/bootloader-zipl/group.yml:11]: I've added tests for this behavior in https://github.com/OpenSCAP/openscap/pull/1551

Comment (ggbecker) [reply linux_os/guide/system/bootloader-zipl/group.yml:11]: That's interesting. Does it mean that rules under this group have automatically the `zipl` CPE check? This was going to be my first question when I first looked to the pull request.

Comment (yuumasato) [reply linux_os/guide/system/bootloader-zipl/group.yml:11]: > Does it mean that rules under this group have automatically the `zipl` CPE check?

In practice yes, I don't know if that is correct behavior though. I couldn't find details about this aspect in the XCCDF Standard.

Comment (yuumasato) [reply linux_os/guide/system/bootloader-zipl/group.yml:11]: What I know is that processing of an item with a platform that doesn't match should be ceased. From XCCDF 1.2, in 7.2.3.3.1 Basics:
> The processing type is Assessing, and the current platform (if known by the product) is not a
member of the set of platforms for this item. If this item is a rule, its result becomes
notapplicable (see Table 26).

This should cause all the Rules in the Group with `zipl` CPE not to be processed.
I'm not sure why the Rules in the Group would evaluate to `notapplicable`.

Comment (ggbecker) [reply linux_os/guide/system/bootloader-zipl/group.yml:11]: I don't understand your concern. It says that if doesn't match the platform and the item is a rule, the result becomes `notapplicable`. Am I missing something?

**Comment by yuumasato** [comment]
Thank you for the through testing.

**Pull request merged**